.travis.yml

```diff
@@ -1,36 +1,33 @@
 language: php
 
-matrix:
-  include:
-    - php: 5.6
-      env: SYMFONY_VERSION=2.0.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.1.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.2.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.3.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.4.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.5.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.6.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.6.* PHPUNIT_VERSION=~3.7
-    - php: hhvm
-      env: SYMFONY_VERSION=2.6.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.7.* PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=2.8.*@dev PHPUNIT_VERSION=~4.0
-    - php: 5.6
-      env: SYMFONY_VERSION=3.0.*@dev PHPUNIT_VERSION=~4.0
+sudo: false
 
-before_script:
+cache:
+  directories:
+    - ~/.composer/cache/files
+
+php:
+  - 5.6
+  - 7.0
+
+env:
+  - SYMFONY_VERSION=2.3.* PHPUNIT_VERSION=^4.0
+  - SYMFONY_VERSION=2.3.* PHPUNIT_VERSION=^5.0
+  - SYMFONY_VERSION=2.7.* PHPUNIT_VERSION=^4.0
+  - SYMFONY_VERSION=2.7.* PHPUNIT_VERSION=^5.0
+  - SYMFONY_VERSION=2.8.* PHPUNIT_VERSION=^4.0
+  - SYMFONY_VERSION=2.8.* PHPUNIT_VERSION=^5.0
+  - SYMFONY_VERSION=3.0.* PHPUNIT_VERSION=^4.0
+  - SYMFONY_VERSION=3.0.* PHPUNIT_VERSION=^5.0
```

Comment:
I would not make this change. This will completely bloat the build matrix as every environment is now applied to each tested PHP version (meaning we talk about n times m jobs now). And by the way you dropped the build that was done on HHVM.